README.md

```diff
@@ -1 +1,83 @@
-Placeholder for Checkstyle Regression Tool readme.
+# Regression-tool
+
+Regression-tool is an automation tool to do regression testing based on proposed patch(Pull Request) 
+of checkstyle project.
+
+## Setup
+
+### Clone
+
+Just clone this repository to your local.

```

Comment:
Expand on this with actual clone command. They can also use their own git utility and specify clone URL.